app/models/form_profile.rb

```diff
@@ -128,7 +135,11 @@ def prefill(user)
     @contact_information = initialize_contact_information(user)
     @military_information = initialize_military_information(user)
     mappings = self.class.mappings_for_form(form_id)
-    form_data = generate_prefill(mappings)
+
+    if FormProfile.prefill_enabled_forms.include?(form_id)
```

Comment:
this wasn't there before and prefills were still being limited, is it needed?

Reply:
My thought in adding it was that `@current_user.can_access_prefill_data?` is used in `app/controllers/in_progress_forms_controller.rb` (https://github.com/department-of-veterans-affairs/vets-api/blob/master/app/controllers/v0/in_progress_forms_controller.rb#L17) to control whether a form is prefilled, but that value always evaluates to `true` (https://github.com/department-of-veterans-affairs/vets-api/blob/master/app/models/user.rb#L150) in the user model. I added this to make sure that a user couldn't pre-fill if the form had pre-filling disabled in that environment. I can take this out though if it's overly restrictive.